_data/samples/widgets.yml

```diff
@@ -0,0 +1,12 @@
+- name: "AppBar"
+  link: "https://docs.flutter.io/flutter/material/AppBar-class.html"
+  image: "<img class='media-object' src='/images/samples/ios_animated_list_very_small.png' alt=''>"
+  description: "Lorem ipsum"
+  detail-page: "/catalog/basic_app_bar/"
```

Comment:
can we qualify this path? we may have other catalogs in the future. qualifying to samples might help.